src/theme/palette.ts

```diff
@@ -1,133 +1,263 @@
-    darker?: string;
-  }
-  interface SimplePaletteColorOptions {
-    darker?: string;
+  interface Interactiveness {
+    default: string;
+    hover: string;
+    disabled: string;
+    pressed: string;
+    secondary: string;
+    tertiary: string;
   }
-  interface Palette {
-    neutral?: Palette['primary'];
-    cultured: {
-      main: string;
-    };
-    actionIcon: {
-      main: string;
+
+  interface PaletteColor {
+    background: {
+      default: string;
+      primary: string;
+      secondary: string;
+      tertiary: string;
       hover: string;
+      blur: {
+        heavy: string;
+        light: string;
+      };
+      brand: Interactiveness;
+      cta: Interactiveness;
+      info: Interactiveness;
+      success: Interactiveness;
+      warning: Interactiveness;
+      error: Interactiveness;
+      code: string;
     };
-    buttonDelete: {
-      main: string;
-      hover: string;
+    text: {
+      default: string;
+      secondary: string;
+      tertiary: string;
+      inverse: string;
+      brand: string;
+      info: string;
+      success: string;
+      warning: string;
+      error: string;
+    };
+    border: {
+      default: string;
+      strong: string;
+      brand: string;
+      normal: string;
+    };
+    icon: {
+      default: string;
+      brand: string;
+      inverse: string;
+      weather: string;
+      disabled: string;
     };
   }
-  interface PaletteOptions {
-    neutral?: PaletteOptions['primary'];
-    cultured?: {
-      main?: string;
+  interface SimplePaletteColorOptions {
+    background: {
+      default: string;
+      primary: string;
+      secondary: string;
+      tertiary: string;
+      hover: string;
+      blur: {
+        heavy: string;
+        light: string;
+      };
+      brand: Interactiveness;
+      cta: Interactiveness;
+      info: Interactiveness;
+      success: Interactiveness;
+      warning: Interactiveness;
+      error: Interactiveness;
+      code: string;
     };
-    actionIcon?: {
-      main?: string;
-      hover?: string;
+    text: {
+      default: string;
+      secondary: string;
+      tertiary: string;
+      inverse: string;
+      brand: string;
+      info: string;
+      success: string;
+      warning: string;
+      error: string;
     };
-    buttonDelete?: {
-      main?: string;
-      hover?: string;
+    border: {
+      default: string;
+      strong: string;
+      brand: string;
+      normal: string;
+    };
+    icon: {
+      default: string;
+      brand: string;
+      inverse: string;
+      weather: string;
+      disabled: string;
     };
   }
 }
 
 export const lightModePalette: PaletteOptions = {
-  primary: {
-    main: CHARCOAL,
-    light: lighten(CHARCOAL, 0.7),
-    contrastText: common.white
-  },
-  secondary: {
-    main: KEPPEL
-  },
   background: {
-    default: accentGrey[90],
-    paper: accentGrey[90]
+    default: Colors.charcoal[100],
+    secondary: Colors.accentGrey[90],
```

Comment:
@nebula-aac :
we need to extend the palette option for secondary and others... under background token, because background type from mui palette has only two option which is default and paper
we can create custom token but i am not getting documentation to extend the already exist type